src/test/airtable/qf.test.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-env mocha */
+
+global.fetch = require('cross-fetch')
+const dotenv = require('dotenv')
+const {
+  clearFundedRecords
+} = require('../../airtable/process_airtable_funding_round_complete')
+dotenv.config()
+const should = require('chai').should()
+const {
+  getWinningProposals,
+  calculateFinalResults,
+  completeEarstructuresValues
+} = require('../../airtable/rounds/funding_rounds')
+
+var fundingRound = {}
+var allProposals = []
+var proposal_csv, expectedResults
+const data = {
+  14: require('./qf_data/round14'),
+  15: require('./qf_data/round15')
```

Comment:
Another good add. I think it's great for us to take static data from airtable to setup these tests.
Seems better encapsulated.